### ISSUE TITLE
(Lombok) annotations with arguments show an error in releases after 1.25.1

### ISSUE THREAD

**Issue by DempseyRoller** [redhat-developer/vscode-java#3454]
[(Lombok) annotations with arguments show an error.]

Type: Bug

##### Environment
- Operating System: Windows 11 / WSL: debian
- JDK version: OpenJDK 17
- Visual Studio Code version: 1.85.1
- Java extension version: 1.25.1+

##### Steps To Reproduce
1. Install latest pre-release
2. Use a lombok annotation with a argument such as @EqualsAndHashcode(callSuper = true)

##### Current Result
Annotations with arguments mess up the annotation reading of VSCode resulting lot's of erros in the editor.

##### Expected Result
Annotations are read as before.

##### Additional Informations
The bug appeared somewhere in the first half of last December. I fixed it today by installing the current stable 1.25.1 version, instead of a prerelease.

**Comment by rgrunber** [comment]
We're aware of this issue in pre-releases. This is a duplicate of https://github.com/eclipse-jdtls/eclipse.jdt.ls/issues/2985 . We're aware of the issue, which is partly why we avoided releasing for so long. We filed at https://github.com/projectlombok/lombok/issues/3564 , and are trying to get a fix into https://github.com/projectlombok/lombok/pull/3565 but it's taking longer than expected.

The problem is this is basically holding us up from doing a release of other nice improvements.
